diff -ur openvcp-web-0.3-rc2/core/openvcp.conf openvcp-web/core/openvcp.conf
--- openvcp-web-0.3-rc2/core/openvcp.conf	2007-09-04 21:19:50.000000000 +0200
+++ openvcp-web/core/openvcp.conf	2007-09-24 00:17:30.000000000 +0200
@@ -51,7 +51,7 @@
 	#
 	# eg.: index.php is in dir /openvcpwebinterface/
 	# set webPath to /openvcpwebinterface
-	webPath openvcp-0.4
+	webPath /openvcp
 	
 	# Support Mail Addy
 	# this will also be used for auto sending mails
@@ -77,7 +77,7 @@
 	db openvcp
 	
 	# Database user
-	user root
+	user openvcp
 	
 	# Database passwort
 	passwd 
